### PR TITLE
Polish localization updates

### PR DESCRIPTION
**Pull request by mavethee**
Adds:
- QEMULauncher string,
- iOS permission strings,
- Remove duplicates (?).

Improves:
- Translation for new features,
- Fix comments and eventual typos.

### CODE REVIEW

**Comment by mavethee** [comment]
...aaaaand I have no clue why it fails apperently 🤡

File: Platform/pl.lproj/Localizable.strings [hunk 1]
```diff
@@ -90,92 +96,94 @@
...
+"The operation is not available." = "Ta operacja jest niedostępna";
+
+/* UTMQemuVirtualMachine.swift */
+"Suspend state cannot be saved when running in disposible mode." = "Stan wstrzymania nie może być zapisany w trybie jednorazowym";
+"Suspend is not supported for virtualization." = "Wstrzymywanie nie jest wspierane dla wirtualizacji.";
+"Suspend is not supported when GPU acceleration is enabled." = "Wstrzymywanie nie jest wspierane, gdy akceleracja grafiki jest włączona.";
+"Suspend is not supported when an emulated NVMe device is active." = "Wstrzymywanie nie jest wspierane, gdy emulowane urządzenie NVMe jest aktywne.";
+"Failed to access data from shortcut." = "Nie udało się uzyskać dostępu do danych ze skrótu.";
+"This build of UTM does not support emulating the architecture of this VM." = "Ta wersja UTM nie wspiera emulacji danej architektury wybranej maszyny wirtualnej.";
+"Failed to access drive image path." = "Nie udało się uzyskać dostępu do ścieżki obrazu dysku."
```
Comment (MMP0) [inline Platform/pl.lproj/Localizable.strings:131]: Missing semicolon. This line causes the build to fail.
```suggestion
"Failed to access drive image path." = "Nie udało się uzyskać dostępu do ścieżki obrazu dysku.";
```

File: Platform/iOS/pl.lpoj/Info-RemotePlist.strings
Comment (MMP0) [inline Platform/iOS/pl.lpoj/Info-RemotePlist.strings:1]: It should be `pl.lproj`.

File: Platform/iOS/pl.lpoj/InfoPlist.strings
Comment (MMP0) [inline Platform/iOS/pl.lpoj/InfoPlist.strings:1]: It should be `pl.lproj`.

**Review by MMP0 — CHANGES_REQUESTED** [review]
It looks like you’re manually adding the files, but `.strings` files need to be compiled, so just putting them won’t get them localized.

To link to the Xcode project, you must check `File Inspector` > `Localization` > `Polish` on the right sidebar.
Sidebar

If a dialog appears, select “Use file”.
Dialog

Also, in the case of `Info-RemotePlist.strings`, it may cause a conflict with the main branch, so please merge the latest `utmapp:main` branch into your branch first.

**Comment by mavethee** [comment]
@MMP0 And that's it? I don't use Xcode sooo yeah, so far all strings I wrote were fine displaying fine, but yeah I don't own an iOS device to check or the AVP. After 'using file' on Polish strings, what needs to be done?

**Comment by mavethee** [comment]
Okay never mind, all it take to upload Xcode file 🤦🏻 
Closing PR was my mistake as wanted to rename branch.
@MMP0 Can you review if you have any more issues to point out?
@osy would it be possible to merge it with stable release as it contains most recent changes in 4.5.x?

**Review by MMP0 — APPROVED** [review]

**Pull request merged**